src/main/java/core/mate/academy/service/MachineService.java

```diff
@@ -16,7 +17,7 @@
      * @param type - any class of Machine sub class. For example: Truck.class or Bulldozer.class
      * @return the list of machines
      */
-    List<Object> getAll(Class type);
+    <E> List<E> getAll(Class type);
```

Comment:
```suggestion
     List<E> getAll(Class type);
```
так як це не статичний метод, його можна окремо не парамтеризовувати